java/build.sh

```diff
@@ -3,12 +3,24 @@ GROUP_ID="com.nvidia.cuvs"
 SO_FILE_PATH="./internal"
 
 if [ -z "$CMAKE_PREFIX_PATH" ]; then
-    export CMAKE_PREFIX_PATH=`pwd`/../cpp/build
+  export CMAKE_PREFIX_PATH=`pwd`/../cpp/build
 fi
 
 cd internal && cmake . && cmake --build . \
-  && cd .. \
-  && mvn install:install-file -DgroupId=$GROUP_ID -DartifactId=cuvs-java-internal -Dversion=$VERSION -Dpackaging=so -Dfile=$SO_FILE_PATH/libcuvs_java.so \
+  && cd ..
```

Comment:
This could be re-written without needing to keep track of what the working directory is with these calls to `cd`, would you please consider that?

```shell
cmake -B ./internal/build -S ./internal
cmake --build ./internal/build
```